File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task7/task7.cpp
// task7.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>
using namespace std;

class Account {
private:
    double balance = 0.0;
public:
    void deposit(double amount) {
        if (amount > 0)
            balance += amount;
        else
            cout << "invalid amount.." << endl;

    }
    double withdraw(double amount) {
        if (amount <= balance)
        {
            balance -= amount;
           // cout << "amount is : " << amount << endl;
            return amount;
        }
        else {
            cout << "Insufficient funds";
            return 0;
        }
    }
    double getBalance() {
        return balance;

    }

};
int main()
{
    Account user1;
   // user1.balance = 1000000;

    user1.deposit(100);
    user1.withdraw(50);

    cout << "user balance is:" << user1.getBalance() << endl;
}

File: Team_Workspace/dalia_ahmed/task7.py

class account:
    def __init__(self):
        self.__balance=0.0
    def deposit(self, amount):
        if amount>0:
            self.__balance += amount
            return self.__balance
        else:
            print("Invalid amount")
    def withdraw(self, amount):
        if amount<=self.__balance:
            self.__balance -= amount
            return self.__balance
        else:
            print("Insufficient funds")
            return 0
        
myaccount=account()
        
# myaccount.__balance = 1000000 dont work 
print(myaccount.withdraw(50))
print(myaccount.deposit(100))


